### PR TITLE
Themes section re-work

### PR DESCRIPTION
**Pull request by jschr**
Preview: https://github.com/jschr/awesome-vscode/tree/themes-section-rework#themes.

- [x] Consistent screenshots for all current themes (with guidelines for creating one)
- [x] Adding [vscodethemes](https://vscodethemes.com/) links for ones that are applicable (themes with a public repo)
- [x] Re-vising the list (removing similar ones / adding some new ones)

### CODE REVIEW

**Comment by viatsko** [comment]
Looks great! If only vscodethemes had a link back to vs code marketplace, I think it would be logical to replace all the marketplace links with vscodethemes links instead.

**Comment by jschr** [comment]
I was debating about adding the marketplace link but opted to leave it out for now because the "Open in VSCode" link takes you to an almost identical page. 

Would you be fine with only some of the themes linking to vscodethemes instead of the marketplace? Only themes with a public repo currently show up.

**Comment by viatsko** [comment]
Sounds good, we can always try to do it this way and gather the feedback on whether there's a friction of not being able to easily access advanced information about the theme right away from a website

**Comment by jschr** [comment]
@viatsko Made all the changes I planned to. Let me know what you think or if you have any suggestions.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ Submit your package in the specific form:
 ![title of the package](link to the .gif file that explains its functionality)
 ```
 
-Failure of complying to these contributing guidelines will result in your pull request being rejected.
+**Submitting a theme**
+
+1.  Submit a link to [vscodethemes](https://vscodethemes.com) if applicable, otherwise use the [marketplace](https://marketplace.visualstudio.com/) link.
+2.  Add your theme in alphabetical order.
+3.  Only one theme per publisher (you may change a featured theme).
+4.  Submit a similar-style screenshot. You may use [this sketch file](https://drive.google.com/file/d/1UpFzk6ZTpD0szySKWcF6r7uageW1mBF_/view) to help (**tip**: line up vscode with the canvas).
```
Comment (jschr) [inline CONTRIBUTING.md]: Not sure where the best place to host files is these days. Happy to upload somewhere else if you have any suggestions.

File: themes/theme-template.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const btn = document.getElementById('btn')
```
Comment (viatsko) [inline themes/theme-template.js]: What is this file for?

File: CONTRIBUTING.md (hunk 1 above)
Comment (viatsko) [reply CONTRIBUTING.md]: Storing files in this git repo sounds reasonable as there's no way we can make all the creators creating consistent screenshots

File: themes/theme-template.js (hunk 2 above)
Comment (jschr) [reply themes/theme-template.js]: This is the file I used in the screenshots—happy to remove if you think it doesn't belong.

Comment (viatsko) [reply themes/theme-template.js]: I think there should be a pointer to this file in CONTRIBUTING.md then

Comment (jschr) [reply themes/theme-template.js]: Mentioned it in CONTRIBUTING.md

**Comment by viatsko** [comment]
Merged. Thank you loads for an amazing work!

**Comment by viatsko** [comment]
Oops, noticed that clicking images doesn't bring you anywhere meaningful, if you want, I can fix it (to match headings links)

**Comment by jschr** [comment]
Ah, yeah that would be a good idea! 

Though that may mean wrapping in an <a> (which is hopefully supported). Using  seemed to be the only way to get the images fit nicely but if that's causing too much trouble I'm totally fine with going back to straight markdown.

Feel free to make the change! I can also do this when I get home later today.

**Pull request merged**